rel-eng/build-packages-for-obs.sh

```diff
@@ -85,6 +85,34 @@ FAILED_PKG=
 VERBOSE=$VERBOSE
 while read PKG_NAME PKG_VER PKG_DIR; do
  for tries in 1 2 3; do
+
+  if [[ $PKG_DIR == *"containers"* ]]; then
```

Comment:
I am not sure why, but it seems this list of changes causes:
```
=== Building container image [proxy-helm]
/tmp/push-packages-to-obs/SRPMS/proxy-helm ~/uyuni
~/uyuni
```
Any idea of why those two lines are printed?

Reply:
This is output of `pushd` and `popd` commands.

Reply:
Can't we hide it?

Reply:
Of course, both prints to `stdout`, so redirect to `>/dev/null` will do.